common/modules/gcp-project/main.tf

```diff
@@ -129,6 +144,7 @@ data "google_iam_policy" "combined" {
     members = [
       "${local.service_accounts}",
       "serviceAccount:${google_service_account.gke_cluster_pod_cert_manager.email}",
+      "serviceAccount:projectowner@${var.project_id}.iam.gserviceaccount.com",
```

Comment:
`projectowner@` is duplicate - once coming from `local.service_accounts` and once explicitly listed above

Reply:
Yup, thanks.